lib/ansible/modules/windows/win_share.ps1

```diff
@@ -111,26 +111,34 @@ Function NormalizeAccounts
 
 $result = @{
     changed = $false
+    actions = @() # More for debug purposes
```

Comment:
If you add this at the very beginning, the output will include an empty list. (Not necessarily a problem if this is on purpose)